test/e2e/utils/test_context.go

```diff
@@ -27,9 +27,7 @@ import (
 	. "github.com/onsi/ginkgo" //nolint:golint
 )
 
-const certmanagerVersion = "v0.11.0"
 const prometheusOperatorVersion = "0.33"
-const certmanagerURL = "https://github.com/jetstack/cert-manager/releases/download/%s/cert-manager.yaml"
 const prometheusOperatorURL = "https://raw.githubusercontent.com/coreos/prometheus-operator/release-%s/bundle.yaml"
```

Comment:
We need to keep this one and then, in the e2e tests for the v3-alpha we should use the upper version and for the v2+ we should keep using this one.